meilisearch-http/Cargo.toml

```diff
@@ -32,17 +32,18 @@ mime = "0.3.16"
 pretty-bytes = "0.2.2"
 rand = "0.7.2"
 rayon = "1.2.0"
+sentry = { version = "0.18.0", features = ["with_rustls", "with_env_logger"] }
```

Comment:
The sentry dependency is really heavy in term of number of added dependencies, isn't there a way to reduce that? (765 additions, that's a lot)

Reply:
I haven't figured out which features to remove.